geode-core/src/test/java/org/apache/geode/internal/process/lang/AvailablePidTest.java

```diff
@@ -100,7 +105,53 @@ public void findAvailablePidsShouldReturnNoDuplicatedPids() throws Exception {
     assertThatNoPidIsDuplicated(availablePid.findAvailablePids(8));
   }
 
-  private void assertThatNoPidIsDuplicated(int[] pids) {
+  @Test(timeout = DEFAULT_TIMEOUT_MILLIS)
+  public void findAvailablePidShouldReturnGreaterThanOrEqualToLowerBound() throws Exception {
+    availablePid = new AvailablePid(new AvailablePid.Bounds(1, 3));
+
+    int pid = availablePid.findAvailablePid();
+
+    assertThat(pid).isGreaterThanOrEqualTo(1);
```

Comment:
I wonder if trying random values for some time period (like the awaitility tests below) might be more robust:

```
availablePid = new AvailablePid(new AvailablePid.Bounds(1, 3));

com.google.common.base.Stopwatch stopwatch = Stopwatch.createStarted();
while (stopwatch.elapsed(TimeUnit.SECONDS) < 10) {
  int pid = availablePid.findAvailablePid();
  assertThat(pid).isLessThanOrEqualTo(3);
}